Lib/pprint.py

```diff
@@ -342,6 +342,32 @@ def _pprint_mappingproxy(self, object, stream, indent, allowance, context, level
 
     _dispatch[_types.MappingProxyType.__repr__] = _pprint_mappingproxy
 
+    def _pprint_simplenamespace(self, object, stream, indent, allowance, context, level):
+        write = stream.write
+        cls_name = object.__class__.__name__
+        if cls_name == "SimpleNamespace":
+            # the SimpleNamespace repr is "namespace"
+            cls_name = "namespace"
+        indent += len(cls_name) + 1
+        delimnl = ',\n' + ' ' * indent
+        items = object.__dict__.items()
+        last_index = len(items) - 1
+
+        stream.write(cls_name + '(')
+        for i, (key, ent) in enumerate(items):
+            write(key)
+            write('=')
+
+            last = i == last_index
+            self._format(ent, stream, indent + len(key) + 2,
```

Comment:
```suggestion
            self._format(ent, stream, indent + len(key) + 1,
```

2 for `len(': ')`, 1 for `len('=')`.